packages/react/src/combobox/ComboboxBase/ComboboxBase.tsx

```diff
@@ -202,14 +215,26 @@ export function ComboboxBase<Option extends DefaultComboboxOption>({
 									<Fragment>
 										{inputItems?.length ? (
 											inputItems.map((item, index) => (
-												<ComboboxListItem
+												<li
+													data-combobox-list-item="interactive"
 													key={`${item.value}-${index}`}
-													isActiveItem={combobox.highlightedIndex === index}
-													isInteractive={true}
 													{...combobox.getItemProps({ item, index })}
+													// Required for Android TalkBack to be able to access the list items
+													// See https://issues.chromium.org/issues/40260928
+													// But stops iOS from being able to access them ◔_◔
+													tabIndex={isIos ? undefined : -1}
 												>
-													{renderItem(item)}
-												</ComboboxListItem>
+													{renderItem ? (
```

Comment:
By not having `renderItem` component as the default arg, whe can speed up a normal use case by only rendering a simple span without all the "slots" for unnecessary things